<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="barra_progresso">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_aumentar">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>Aumentar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_zerar">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(204, 0, 0);</string>
   </property>
   <property name="text">
    <string>Zerar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
